homeassistant/components/arwn/sensor.py

```diff
@@ -109,11 +117,14 @@ def async_sensor_event_received(msg):
 class ArwnSensor(Entity):
     """Representation of an ARWN sensor."""
 
-    def __init__(self, name, state_key, units, icon=None):
+    def __init__(self, topic, name, state_key, units, icon=None):
         """Initialize the sensor."""
         self.hass = None
         self.entity_id = _slug(name)
         self._name = name
+        # This mqtt topic for the sensor which is its uid
+        self._uid = slugify(topic)
```

Comment:
Is there a reason we need to slugify this?

Reply:
Good point, I wasn't sure if it handled / and space correctly, but it seems to.